src/components/ec-table-head/ec-table-head.vue

```diff
@@ -101,6 +120,10 @@ export default {
   &__cell-wrapper {
     display: flex;
     align-items: center;
+
+    &--justify-end {
```

Comment:
we have `--text-center` and `--text-right` in cells, we should have the same naming here. But first check with Uve if this is actually desired behaviour.

Reply:
Changed to `--is-type-icon` and `--is-type-currency`